mappings/net/minecraft/entity/passive/AbstractDonkeyEntity.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/unmapped/C_javurlgg net/minecraft/entity/passive/AbstractDonkeyEntity
+	FIELD f_jyjkqgji INVENTORY_CHEST_COUNT I
```

Comment:
Chest count? is this the amount of slots?

Reply:
I believe so? I can't find usage in code so I'm not sure

Reply:
```suggestion
	FIELD f_jyjkqgji INVENTORY_SLOT_COUNT I
```

Reply:
Agree with Matrix